packages/remix-dev/vite/plugin.ts

```diff
@@ -611,6 +611,7 @@ export const remixVitePlugin: RemixVitePlugin = (options = {}) => {
                   plugin !== null &&
                   "name" in plugin &&
                   plugin.name !== "remix" &&
+                  plugin.name !== "remix-react-refresh-babel" &&
```

Comment:
I just realized `"remix-react-refresh-babel"` already mostly no-op in child compiler because `transform` is called with `ssr = true`

https://github.com/remix-run/remix/blob/fd8d3a25ce17b6969081991db39990247f8c051c/packages/remix-dev/vite/plugin.ts#L221

https://github.com/remix-run/remix/blob/fd8d3a25ce17b6969081991db39990247f8c051c/packages/remix-dev/vite/plugin.ts#L1027-L1030

So, I don't think this would affect perf in any meaningful way, but it may be still considered as refactoring.
Please feel free to decide whether this tiny cleanup would be worth included. Closing this PR is totally fine with me.

Reply:
Thanks for looking into this! As you say, I think it's better to leave the code as-is.